services/web-server/src/login/strategies/mozilla-auth0.js

```diff
@@ -240,6 +239,7 @@ module.exports = class MozillaAuth0 {
       (request, response) => {
         response.render('callback', {
           user: request.user,
+          publicUrl: cfg.app.publicUrl,
```

Comment:
I don't think this config exists -- I don't see `publicUrl` in `config.yml` anyway, and I recall getting rid of that configuration a while ago.  I think you'll want `cfg.taskcluster.rootUrl`.

Reply:
`publicUrl` is used for the callback URL https://github.com/taskcluster/taskcluster/blob/6ccceca715cdcad390af530119a3155f8a41a4ee/services/web-server/src/login/strategies/mozilla-auth0.js#L206 The taskcluster-web-server service in heroku has `TASKCLUSTER_ROOT_URL` pointing to `https://taskcluster.net`. If we send a post message to this URL, I think the front end won't be able to intercept the response since it's a different origin.

Reply:
>  I don't see publicUrl in config.yml

https://github.com/taskcluster/taskcluster/blob/6ccceca715cdcad390af530119a3155f8a41a4ee/services/web-server/config.yml#L4

Reply:
Whoops, I looked in `services/worker-manager/config.yml` since that's what I was working on.  You're right about publicUrl.